coral/src/app/features/topics/schema-request/components/TopicSchema.test.tsx

```diff
@@ -0,0 +1,192 @@
+import { cleanup, screen } from "@testing-library/react/pure";
+import userEvent from "@testing-library/user-event";
+import {
+  TopicSchema,
+  readFile,
+} from "src/app/features/topics/schema-request/components/TopicSchema";
+import { renderForm } from "src/services/test-utils/render-form";
+import { z } from "zod";
+import { TopicRequestFormSchema } from "src/app/features/topics/schema-request/utils/zod-schema";
+
+jest.mock("src/app/features/topics/schema-request/utils/read-file");
+const mockReadFiles = readFile as jest.MockedFunction<typeof readFile>;
+
+// Temp mock for the editor, based on this:
+// https://github.com/suren-atoyan/monaco-react/issues/88#issuecomment-887055307
+// github issue to investigate a reusable testing approach:
+// 🐙 https://github.com/aiven/klaw/issues/475
+jest.mock("@monaco-editor/react", () => {
```

Comment:
I tried to make a global mock for this in [mock-monaco-editor.tsx](https://github.com/aiven/klaw/blob/main/coral/test-setup/mock-monaco-editor.tsx). Did you try to use that?

Reply:
Oh! I've seen the mock through searching and based mine on that, but didn't realise it was it's own file. 

Will update my setup & also, I checked the setup time, because now we're now importing React in all our test files, but surprisingly the impact is not really visible - and now I realise we're not importing `react` but `React` (the type, from the @types package) which explains that 💡  And that also means we can do that for the Icon mock, too, if we want to 👯